pkg/service/project.go

```diff
@@ -79,7 +85,23 @@ func (s *projectService) Update(ctx context.Context, req *v1.ProjectUpdateReques
 func (s *projectService) Delete(ctx context.Context, req *v1.ProjectDeleteRequest) (*v1.ProjectResponse, error) {
 	project := req.NewProject()
 	err := s.projectStore.Delete(ctx, project.Meta.Id)
-	return project.NewProjectResponse(), err
+	if err != nil {
+		return nil, err
+	}
+	filter := map[string]any{
+		"projectmember ->> 'project_id'": project.Meta.Id,
+	}
+	memberships, _, err := s.projectMemberStore.Find(ctx, filter, nil)
+	if err != nil {
+		return nil, err
+	}
+	for _, m := range memberships {
+		err := s.projectMemberStore.Delete(ctx, m.Meta.Id)
```

Comment:
Would it be possible to just delete these member through a single SQL query? I guess it's more robust than a loop.

Reply:
This requires a additional Store Interface `DeleteAll` or so ?

Reply:
Don't know how difficult this is. Maybe we can return "Unimplemented" for project and tenant and just implement for the member stores?

Reply:
i have i implementation, unsure if this should be pushed to this PR or create a new one

Reply:
pushed here

Reply:
done